building-artifacts/controlling-aspects-of-the-build-command/use-of-multiple-config-file-in-build-command.md

```diff
@@ -24,7 +24,7 @@ For example, if you have two packages, package 1 is dependent on `SharedActiviti
     }
   ],
   "plugins": {
-    "sfpowerscripts":{
+    "sfp":{
```

Comment:
@ruslan-kurchenko  let me check the code, I think its still sfpowerscripts for this one, for backward compatibility